### PR TITLE
Ahsaask/terms condition

### PR DESCRIPTION
**Pull request by Ahsaask**
- Fixed UI for Terms and condition based on figma 
- Auto save in ideaform: using local-storage  
- fixed some minor bugs in idea submission form
- added back-button to terms and condition
- added logic to back button to exit tab when no back url available
- added responsiveness to terms and condition

Issues: #1269 #1952 

@dbradham @chadcrotchett @AlinaU1

### CODE REVIEW

File: staging/kustomization.yaml
Comment (Ahsaask) [inline staging/kustomization.yaml:1]: I did not change this file, not sure why its here. (possibly came when i merged it with development/ideaspace)

Comment (dbradham) [reply staging/kustomization.yaml:1]: @Ahsaask looks like this is from the linter tool, it just added some whitespace and changed double quotes to single quotes. Should be fine to merge.

File: apps/ideaspace/src/components/common/BackButton/BackButton.js [hunk 1]
```diff
@@ -8,8 +8,10 @@ const BackButton = ({ buttonType, clickHandler, backRoute }) => {
   const goBack = () => {
     if (backRoute) {
       router.push(backRoute);
+    } else if (window.history.length > 1) {
+      window.history.back();
     } else {
-      window.history.back(-1);
+      window.close();
```
Comment (dbradham) [inline apps/ideaspace/src/components/common/BackButton/BackButton.js:14]: is closing the current window when users navigate part of the requirements for this change? @Ahsaask @chadcrotchett

Comment (Ahsaask) [reply apps/ideaspace/src/components/common/BackButton/BackButton.js:14]: Requirement: I want to be able to navigate back from the 'terms and conditions' page to the previous page I visited.

We decided that when the user navigates to a new tab for idea-space terms condition, we will close the tab to prompt user to get back to the submission form. Ideally this should not replace any other functionalities. Let me know if there are some concerns.

File: staging/kustomization.yaml
Comment (chadcrotchett) [reply staging/kustomization.yaml:1]: @dbradham Is it an option to 'linter' the whole code base in one PR so these whitespace formatting items don't distract future PRs?

**Pull request merged**